.github/workflows/release.yml

```diff
@@ -1,9 +1,13 @@
-name: Generate Static Assets
+name: Release
 
 on:
-  push:
-    branches:
-      - "release/**"
+  workflow_dispatch:
+    inputs:
+      version:
```

Comment:
I don't see this input being used anywhere in the workflow?

Reply:
Ah yeah, thanks! I was going to put it in the commit message and also use for future work when this workflow is able to generate CHANGELOG and update the `version.go` file.